x-pack/platform/plugins/shared/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -71,7 +72,9 @@ export const AllCasesList = React.memo<AllCasesListProps>(
         }
 
         for (const assignee of caseInfo.assignees) {
-          acc.add(assignee.uid);
+          if (!isEmpty(assignee.uid)) {
+            acc.add(assignee.uid);
```

Comment:
What do you think of moving the responsibility to the `useBulkGetUserProfilesMock` hook?

Reply:
We'll add it to `bulkGetUserProfiles` in `x-pack/platform/plugins/shared/cases/public/containers/user_profiles/api.ts`